app/assets/javascripts/hydra-editor/editMetadata.js

```diff
@@ -1,3 +1,3 @@
 Blacklight.onLoad(function() {
-  $('.multi_value.form-group').manage_fields();
+  $('.repeating-field.form-group').manage_fields();
```

Comment:
@jechols why the change in css class? This would break current implementations, I believe.